server/app.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const db = require('./db/index.js');
+
+const app = express();
+
+app.set('port', 3004);
+
+//app.use(express.static(__dirname + '' ));
+/// returns all alsovieweditems
+app.get('/api/alsovieweditems', (req, res) => {
+
+  var queryString = 'select * from alsovieweditems';
+  var queryArgs = [];
+
+  db.query(queryString, queryArgs, function(err, results) {
+    if (err) {
+      console.log(err);
+      res.status(500).send();
+    } else {
+      res.status(200).json(results);
+    }
+  });
+
+});
+
+/// returns all categoryids assigned to items
+app.get('/api/alsovieweditems/categoryids', (req, res) => {
+  var queryString = 'select categoryid from alsovieweditems';
+  var queryArgs = [];
+
+  db.query(queryString, queryArgs, function(err, results) {
+    if (err) {
+      console.log(err);
+      res.status(500).send();
+    } else {
+      var categoryids = [];
+      for (var i = 0; i < results.length; i++) {
```

Comment:
The style guide  opposes the use of the ++ incrementing of variables.

37 can be written like
 `for (let i = 0; i < results.length; i += 1) {`